packages/web-components/src/components/cta-block/cta-block-item.ts

```diff
@@ -60,6 +60,11 @@ class DDSCTABlockItem extends StableSelectorMixin(DDSContentItem) {
     const { name } = target as HTMLSlotElement;
     const hasContent = (target as HTMLSlotElement)
       .assignedNodes()
+      .filter(elem =>
+        (elem as HTMLElement).matches !== undefined
+          ? !(elem as HTMLElement).matches((this.constructor as typeof DDSCTABlockItem).selectorButtonGroup)
+          : false
+      )
```

Comment:
```suggestion
      .filter(elem => !(elem as HTMLElement).matches?.((this.constructor as typeof DDSCTABlockItem).selectorButtonGroup))
```

instead of checking if `elem.matches()` is defined would we be able to make use of optional chaining somehow?

Reply:
You're right, that method works and looks nicer as well. I've noticed some files still use the current logic, so I might open a new PR updating those as well. Thanks @emyarod!